checks/junit.go

```diff
@@ -70,22 +77,49 @@ func (c *JunitChecker) Check(extConfig external.Check) *pkg.CheckResult {
 	if junitCheck.GetDisplayTemplate() != "" {
 		textResults = true
 	}
+	interval := junitCheck.GetInterval()
+	timeout := junitCheck.GetTimeout()
 	var junitStatus JunitStatus
 	template := junitCheck.GetDisplayTemplate()
 	pod := &corev1.Pod{}
 	pod.APIVersion = corev1.SchemeGroupVersion.Version
 	pod.Kind = podKind
+	pod.Labels = map[string]string{
+		junitCheckSelector: junitCheckLabelValue,
+	}
 	if junitCheck.GetNamespace() != "" {
 		pod.Namespace = junitCheck.GetNamespace()
 	} else {
 		pod.Namespace = corev1.NamespaceDefault
 	}
 	if junitCheck.GetName() != "" {
-		pod.Name = junitCheck.GetName()
+		pod.Name = junitCheck.GetName() + "-" + strconv.Itoa(int(start.Unix()))
```

Comment:
This will be an epoch - which isn't really useful, either use `yyyymmddhhmm` or random 4 - 6 chars